Makefile

```diff
@@ -21,7 +21,7 @@ verify-go-bindata-installation:
 
 update-bindata: verify-go-bindata-installation
 	find templates/ -type f -exec perl -i -pe "chomp if eof" {} \;
-	go-bindata -pkg cloudconfig templates/...
+	go-bindata -modtime 1 -mode 420 -pkg cloudconfig templates/...
```

Comment:
We do this only to make `bindata.go` deterministic, it doesn't matter which timestamp we use, but it has to be the same always.